sdk/eventgrid/azure-messaging-eventgrid-cloudnative-cloudevents/src/test/java/com/azure/messaging/eventgrid/cloudnative/cloudevents/EventGridCloudNativeEventPublisherTests.java

```diff
@@ -84,6 +88,46 @@ public void publishEventGridEvents() {
         EventGridCloudNativeEventPublisher.sendEvents(egClient, cloudEvents);
     }
 
+    @Test
+    public void publishEventGridEventsToDomain() {
+        // When publishing to an Event Grid domain with cloud events, the cloud event source is used as the domain topic.
+        // The Event Grid service doesn't support using an absolute URI for a domain topic, so you would need to do
+        // something like the following to integrate with the cloud native cloud events:
+        builder.endpoint(System.getenv("AZURE_EVENTGRID_CLOUDEVENT_DOMAIN_ENDPOINT"))
+            // Event Grid Domain endpoint with CloudEvent Schema
+            .credential(new AzureKeyCredential(System.getenv("AZURE_EVENTGRID_CLOUDEVENT_DOMAIN_KEY")));
```

Comment:
This change is causing pipelines to fail since the environment variable isn't configured